SpringProject/robot_controller_v3.m

```diff
@@ -128,6 +128,7 @@
 Kp_floor = -5000; % floor spring constant
 Kd_floor = -800; % floor damping coefficient
 b_fric_floor = -2000; % floor coefficient of lateral, viscous friction
+state = zeros(1,36);
```

Comment:
and here

Reply:
fixed in ce27c1abd3bb5b6b2af4460e46af67cfc407ce39

Reply:
Fixed in 7d35f0b138ebeaaf6bb62ae2fa4fc652f9b7e359